pkg/pb/metainfo.proto

```diff
@@ -103,4 +104,15 @@ message ListSegmentsResponse {
       
     repeated Item items = 1;
     bool more = 2;
+}
+
+message ValueAttributionRequest{
+    bytes bucket_id = 1;
+    bytes user_id = 2;
```

Comment:
I don't think we need the user_id here. We should be pulling the user_id out of the user making the request.

Reply:
In fact we need only `bucket_name` (bucket_id) and `partner_id`. `BucketID` is a `project id` + `bucket name` and partner won't know our internal project id. With api keys and bucket name we are able to find coresponding bucket.